docs/getting-started.md

```diff
@@ -128,20 +201,93 @@ If you prefer to configure the service with commandline flags you can see the av
 : Token to grant metrics access, empty default value
 
 --debug-pprof
-: Enable pprof debugging, defaults to `false`
+: Enable pprof debugging
 
 --debug-zpages
-: Enable zpages debugging, defaults to `false`
+: Enable zpages debugging
 
 --http-addr
 : Address to bind http server, defaults to `0.0.0.0:9130`
 
---http-namespace
-: Namespace for internal services communication, defaults to `com.owncloud.web`
-
 --http-root
 : Root path of http server, defaults to `/`
 
+--http-namespace
+: Set the base namespace for service discovery, defaults to `com.owncloud.web`
+
+--identity-manager
+: Identity manager (one of ldap,kc,cookie,dummy), defaults to `ldap`
+
+--transport-tls-cert
+: Certificate file for transport encryption, empty default value
+
+--transport- tls-key
+: Secret file for transport encryption, empty default value
```

Comment:
The default value is misleading here, as we create a key+cert combo. I'd take `empty default value` off and explain if not provided a development pair is used instead